letter_opener_web.gemspec

```diff
@@ -17,18 +17,12 @@ Gem::Specification.new do |gem|
 
   gem.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
   gem.executables   = gem.files.grep(%r{^exe/}).map { |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
   gem.add_dependency 'actionmailer', '>= 5.2'
   gem.add_dependency 'letter_opener', '~> 1.7'
```

Comment:
[FWIW - 1.9.0 requires ruby 3.0+!](https://github.com/ryanb/letter_opener/blob/master/CHANGELOG.md#190)

Reply:
Good to know. Now that this PR has CI ✅ again, I'm thinking maybe I'll merge it. Then take a pass at dropping older Rubies, updating minimum dependencies, etc… in another PR. Just to keep the PRs smaller and easier to "see" the deltas, if nothing else.